docs/configuration.md

```diff
@@ -392,6 +392,16 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
+  <td><code>spark.reducer.maxReqsInFlight</code></td>
+  <td>Int.MaxValue</td>
+  <td>
+    This configuration limits the number of remote blocks to fetch at any given point.
```

Comment:
nit: `the number of remote blocks to fetch` -> `the number of remote requests to fetch blocks`